### ISSUE TITLE
Multiple time texts in a speech command are not correctly spoken by Homey

### ISSUE THREAD

**Issue by corvdniet** [athombv/homey#2189]
Multiple time texts in a speech command are not correctly spoken by Homey

**Metadata**

* **Homey Version**: 1.5.6-rc.8
* **Homey Log Code**: -
* **Smartphone Platform**: -
* **Smartphone App Version**: -

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

Tried to use multiple time texts in a "say something" flow command

**What did you expect as result?**

That the whole sentence would be spoken by Homey

**What was the result?**

When using multiple time texts in a "say something" command Homey skips parts of words (words following the time part of the sentence). The first time text and following words go correct. Words behind a second, third, etc. time part of the sentence are partially "eaten up" by Homey. 
Example1: 01:01 test 1 , 01:02 test 2 is spoken by Homey as: 1 uur 1 test 1 , 1 uur 2 st 2
Example1: 10:01 test 1 , 10:02 test 2 is spoken by Homey as: 10 uur 1 test 1 , 10 uur 2 t 2 

**Any other remarks**
Added file with examples
[Bug speech time.txt](https://github.com/athombv/homey/files/1678302/Bug.speech.time.txt)

**Comment by bvdbos** [comment]
Duplicate of #1624 or #1297 or #1627 ? Does a reboot fix this or is this consistent capping of strings?

**Comment by corvdniet** [comment]
Nope. I have stutters as well (sometimes), but this behaviour is different. Don't think the latest RC introduced this issue, by the way.

**Comment by Dijker** [comment]
I guess it is because it is developed in [Twente](https://nl.m.wikipedia.org/wiki/Twents_(dialect) ) 
They don't speak out everything we write... 😁
